config/initialize.yaml

```diff
@@ -5,10 +5,26 @@ Iyy : 16.655602e-6
 Izz : 29.261652e-6
 d: 4e-3
 cft: 0.005964552 
+drag: 0 # 0: do not include drag model, 1: include simple drag model
+## Payload Properties
+payload: 0 # Payload flag, 0: no payload, 1: payload
+m_p: 0.01 # Mass payload [kg]
+l_c: 0.7  # Length of cable [m]
+## INITIAL CONDITIONS FOR PAYLOAD
+xl  :  0
+yl  :  0
+zl  :  0
+vxl :  0
+vyl :  0
+vzl :  0
+px  :  0  # Initial condition for directional unit vector from UAV to payload
+py  :  0  # p = [0,0,-1]
+pz  :  -1  # pointing downwards expressed in Inertial Frame
+wxl :  0
+wyl :  0 # Angular velocity of payload
+wzl :  0
```

Comment:
It would make sense to use the hierarchical feature of yaml:

```
payload:
   mode: "enabled"
   m_p: 0.01 # Mass payload [kg]
   ...
```

Reply:
I did not know that this exists in yaml files. I will change it